addons/payment_adyen/models/payment.py

```diff
@@ -153,7 +154,7 @@ def adyen_form_generate_values(self, values):
                 'merchantReturnData': json.dumps({'return_url': '%s' % values.pop('return_url')}) if values.get('return_url') else False,
                 'merchantSig': self._adyen_generate_merchant_sig('in', values),
             })
-
+        _logger.info('adyen_form_generate_values: %s' % pprint.pformat(values))
```

Comment:
That's actually a good idea, no need to filter the data here since all the data sent to the render method should be visible to the customer (in the HTML form).
I am still going to check if we don't put secret values for each acquirer.